main.css

```
@@ -496,7 +493,13 @@ footer {
 }
 
 /*Адаптив*/
-
+	
+@media (min-height: 501px) and (max-height: 1499px) and (orientation: landscape) {
```

nit: давай поменяем максимальную высоту на 1000px, чтобы на слишком больших экранах Projects слишком далеко не задвигать?

```suggestion
@media (min-height: 501px) and (max-height: 1000px) and (orientation: landscape) {
```